python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -138,9 +138,8 @@ def checkHostName():
         except:
             pass
 
-        self.syscfg.svo.stopService("cloudstack-management")
-
         if self.syscfg.env.noStart == False:
+            self.syscfg.svo.stopService("cloudstack-management")
```

Comment:
so if nostart is false (start is true ??) the service is stopped. Am I understanding this wrong? it seems like we don't want this here and not unconditionally either but only if nostart is true.